### DISCUSSION TITLE
Welcome to OpenAM Discussions!

### DISCUSSION THREAD

**Discussion by maximthomas** [OpenIdentityPlatform/OpenAM#332] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by FireBurn** [comment]
Hi. I was just wondering what the plans were for migrating away from the old sun.* classes. Is there a plan to move to building with Java 11?

I guess more generally is there a project roadmap?

**Reply by maximthomas** [reply]
Hi @FireBurn. 
Yes, we have plans to build OpenAM with Java 11. 
I think it is a good idea to create a project roadmap within the community. We appreciate any suggestions on what should be implemented in the recent releases.
